src/redux/Categories/Categories.js

```diff
@@ -0,0 +1,17 @@
+const CHECK_STATUS = 'react-bookstore-app/categories/CHECK_STATUS';
+const initialState = {};
+
+// Reducer
+export default function reducer(state = initialState, action) {
+  switch (action.type) {
+    // do reducer stuff
+    case CHECK_STATUS:
+      return { ...state };
```

Comment:
- [ ] Kindly on the `CHECK_STATUS` action you have to return `Under construction` as the requirements say.